doc/input-output-reference/src/overview/group-air-distribution-equipment.tex

```diff
@@ -121,7 +128,14 @@ \subsubsection{Inputs}\label{inputs-1-001}
 
 \subsubsection{Outputs}
 
-There are no outputs for the constant volume NoReheat air terminal.
+\begin{itemize}
+\item
+  HVAC,Average,Zone Air Terminal Outdoor Air Volume Flow Rate {[}m3/s{]}
+\end{itemize}
+
+\paragraph{Zone Air Terminal Outdoor Air Volume Flow Rate {[}m3/s{]}}\label{zone-air-terminal-outdoor-air-volume-flow-rate-m3s}
+
+This output is the amount of outdoor air entering the zone. This is the average value over the frequency being reported. The amount of outdoor air is defined as the terminal unit air volume flow rate multiplied by the fraction of outdoor air entering the air loop's outside air system.
 
```

Comment:
This is missing description of existing report variable for AirTerminal:SingleDuct:ConstantVolume:NoReheat air terminal unit.